### ISSUE TITLE
Error: Command failed @ Centos-7

### ISSUE THREAD

**Issue by piecioshka** [piecioshka/encoding-checker#3]
I'm getting the same error on Centos-7 and I do have the `find` command.

![image](https://user-images.githubusercontent.com/5749465/44679134-1a26a280-aa08-11e8-958a-d335220583f0.png)

Any idea where the problem may stem from? 

One possible issue may be the fact that I do not have root access. Therefore, I got IT to install `npm` globally on the cluster. Then, I followed the advice from [this website](https://docs.appcelerator.com/platform/latest/?mobile=/guide/Installing_Node) and set `npm` to install (new) programs to my home directory by adding the following to my `.bash_profile`: `export NPM_CONFIG_PREFIX=$HOME` and then `source .bash_profile`.

I noticed that your instructions for installing `encoding-checker` with `npm` used the `-g` flag which is for global installation. When I ran it as is it seems to work as it can access the `encoding-checker` program but fails when I run `encoding-checker -d .` (see above image).

When I use the `-l` local flag instead I get this error:

```
moldach@cortex:~ $ npm install -l encoding-checker

│ /exports/meaney.lab/moldach
│
└─┬ encoding-checker@1.0.1
  │ Tool to investigate files with different encoding than passed
  │ git+ssh://git@github.com/piecioshka/encoding-checker.git
  │ https://github.com/piecioshka/encoding-checker#readme
  ├── colors@1.3.2
  │   get colors in your node.js console
  │   git+ssh://git@github.com/Marak/colors.js.git
  │   https://github.com/Marak/colors.js
  └── commander@2.17.1
      the complete solution for node.js command-line programs
      git+https://github.com/tj/commander.js.git
      https://github.com/tj/commander.js#readme

npm WARN enoent ENOENT: no such file or directory, open '/exports/meaney.lab/moldach/package.json'
npm WARN moldach No description
npm WARN moldach No repository field.
npm WARN moldach No README data
npm WARN moldach No license field.
```
Any idea what I should try? Thanks!

_Originally posted by @moldach in https://github.com/piecioshka/encoding-checker/issues/2#issuecomment-416333824_

**Comment by piecioshka** [comment]
https://github.com/piecioshka/encoding-checker/pull/10

**Comment by piecioshka** [comment]
hi @moldach !
Please tell me that this version fix your problem with error: "Command failed".

**Comment by piecioshka** [comment]
Problem should be solved in v1.1.14
